<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTransaction</class>
 <widget class="QDialog" name="addTransaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date &amp; Time</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Additional Info</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QDateTimeEdit" name="dte_time_date"/>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="cb_category"/>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cb_transaction"/>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QLineEdit" name="le_info"/>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="dsb_amount"/>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Transaction Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dte_time_date</tabstop>
  <tabstop>cb_category</tabstop>
  <tabstop>cb_transaction</tabstop>
  <tabstop>dsb_amount</tabstop>
  <tabstop>le_info</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
